test/utilities/FormManager.test.tsx

```diff
@@ -54,4 +56,33 @@ describe('FormManager', () => {
     const formManager = await getFormManager(fieldSets, { id, name, notId });
     expect(formManager.formModel).toEqual({ name, id });
   });
+
+  it(`Correctly maintains id`, async () => {
+    const fieldSets = [[{ field: 'name' }]]
+      , name = faker.name.firstName()
+      , id = faker.random.uuid()
+      , notId = faker.random.uuid();
+
+    const formManager = await getFormManager(fieldSets, { id, name, notId });
+    expect(formManager.formModel).toEqual({ name, id });
+  });
+
+  it(`Handles 500 responses`, async () => {
+    const fieldSets = [[{ field: 'name' }]]
+      , name = faker.name.firstName()
+      , id = faker.random.uuid()
+      , notId = faker.random.uuid()
+      , response = {
+        config: {},
+        data: '<!DOCTYPE html><html lang="en"><head /></html>',
```

Comment:
Why include all this extra response information when it's only checking status?

Reply:
I copied it directly from a real response. This string here is unexpected by the code so I wanted it to be there. I realize it _shouldn't_ affect anything but hey that's what tests are for, right?